### ISSUE TITLE
Add `helm status [APP_NAME] --validate` flag

### ISSUE THREAD

**Issue by viglesiasce** [kubernetes/helm#1498]
What I'd like this flag to do is print out a templatized validation script that can be run in order to ensure that the application is actually running properly. I could see something like this:

```shell
$ helm install --name test jenkins
$ helm status test --validate | bash
[OK] At least one endpoint found
[OK] Jenkins server running
[OK] Able to access /login URL
```

The validation script would live with the chart much like NOTES.txt. This would make CI functional validation much much easier and would also allow users to self validate their running charts periodically.

**Comment by prydonius** [comment]
cc/ @michelleN

**Comment by erictune** [comment]
If we have this, I could think about running all charts against head kubernetes.

**Comment by viglesiasce** [comment]
Tasty idea @erictune. I think this would really kick things up a notch in terms of how we can scale the charts repo and the testing of helm & kubernetes.

**Comment by technosophos** [comment]
I believe this is now closed with the `helm test` function in 2.2.0. Please re-open if I misunderstood this.
